spek-runtime/src/nativeMain/kotlin/org/spekframework/spek2/launcher/console.kt

```diff
@@ -0,0 +1,11 @@
+package org.spekframework.spek2.launcher
+
+actual class ConsoleLauncher: AbstractConsoleLauncher() {
+    override fun parseArgs(args: List<String>): LauncherArgs {
+        return LauncherArgs(
+            listOf(ConsoleReporterType(ConsoleReporterType.Format.BASIC)),
```

Comment:
@charleskorn eventually we need to parse `args` here, but for now lets send the default emulating what you had before.

Reply:
Sounds good to me.